table/table_test.go

```diff
@@ -9,16 +9,22 @@ import (
 
 	"errors"
 	"io"
-	"io/ioutil"
 
 	"github.com/ONSdigital/dp-api-clients-go/zebedee"
 	"github.com/ONSdigital/dp-file-downloader/table"
 	"github.com/ONSdigital/dp-file-downloader/table/testdata"
 	. "github.com/smartystreets/goconvey/convey"
 )
 
-const contentHost = "content"
-const renderHost = "render"
+var requestURI = "/foo/bar.json"
+var requestFormat = "html"
+var expectedDisposition = "attachment; filename=\"bar.html\""
+var accessToken = "myAccessToken"
+var uriParam = "&uri="
+var expectedContentType = "text/html"
+var expectedContent = "renderServerResponse"
+var contentServerResponse = "contentServerResponse"
+var baseURL = "http://localhost/download/table?format="
```

Comment:
Could be a var block or use const for these but not massively fussed.

Reply:
I may as well do a var block - keeps it all nice and tidy